airgun/views/host.py

```diff
@@ -466,26 +471,77 @@ class additional_information(SatTab):
+            './/div[contains(@class, "pf-c-tabs")]//ul'
+            "/li[button[normalize-space(.)={@tab_name|quote}]]"
+        )
+        ROOT = '//section[@id="generalSection"]'
+
+        orgnization = FormSelect('OUIA-Generated-FormSelect-default-1')
+        location = FormSelect('OUIA-Generated-FormSelect-default-2')
+        host_group = FormSelect('OUIA-Generated-FormSelect-default-3')
+        operating_system = FormSelect('OUIA-Generated-FormSelect-default-4')
+        linux_host_init_link = Link('//a[text()="Linux host_init_config default"]')
+        capsule = FormSelect('OUIA-Generated-FormSelect-default-5')
+        insecure = Checkbox(id='reg_insecure')
+
+    @View.nested
+    class advanced(Tab):
+        TAB_NAME = 'Advanced'
+        TAB_LOCATOR = ParametrizedLocator(
+            './/div[contains(@class, "pf-c-tabs")]//ul'
+            "/li[button[normalize-space(.)={@tab_name|quote}]]"
+        )
+        ROOT = '//section[@id="advancedSection"]'
+
+        setup_rex = FormSelect('OUIA-Generated-FormSelect-default-6')
+        setup_insights = FormSelect('OUIA-Generated-FormSelect-default-7')
+        install_packages = TextInput(id='reg_packages')
+        repository = TextInput(id='reg_repo')
+        repository_gpg_key_url = TextInput(id='reg_gpg_key_url')
+        token_life_time = TextInput(id='reg_token_life_time_input')
+        rex_interface = TextInput(id='reg_rex_interface_input')
+        activation_keys = BaseMultiSelect('OUIA-Generated-Select-typeaheadmulti-1')
+        life_cycle_env = FormSelect('OUIA-Generated-FormSelect-default-8')
+        ignore_error = Checkbox(id='reg_katello_ignore')
+        force = Checkbox(id='reg_katello_force')
 
     @property
     def is_displayed(self):
-        breadcrumb_loaded = self.browser.wait_for_element(self.breadcrumb, exception=False)
-        return (
-            breadcrumb_loaded
-            and self.breadcrumb.locations[0] == 'Registrations'
-            and self.breadcrumb.read() == 'Register Host'
+        return self.browser.wait_for_element(self.general.operating_system, exception=False)
+
+    def before_fill(self, values):
+        """Fill some of the parameters tab widgets with values.
+
+        Args:
+            values: A dictionary of ``widget_name: value_to_fill``.
+
+        Note:
+            Some of the fields are disabled for few seconds and get enabled based
+            on the selection, handled separately
+        """
+        for field in ('host_group', 'operating_system'):
+            field_value = values.get('general').get(field)
+            if field_value:
+                wait_for(
+                    lambda: self.general.__getattribute__(field).is_enabled,
+                    timeout=30,
+                    delay=2,
+                    logger=self.logger,
+                )
+                self.general.__getattribute__(field).fill(field_value)
+        wait_for(
+            lambda: self.general.linux_host_init_link.is_displayed,
```

Comment:
Same here  as above for default selenium wait behavior ^

Reply:
as mentioned in the earlier comment https://github.com/SatelliteQE/airgun/pull/587#discussion_r674538108